man/systemd.network.xml

```diff
@@ -925,6 +925,21 @@ Table=1234</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IPv4ProxyARPPrivateVLAN</varname></term>
```

Comment:
please suffix the option name with `=`. We do that in our docs and log messages for all options, to make clear they are options

Reply:
Gotcha!

Reply:
Done.